lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart

```diff
@@ -173,11 +169,31 @@ class VerifyBackupPhrasePageState extends State<VerifyBackupPhrasePage> {
   _selectIndexes() {
     List list = List.generate(23, (i) => i);
     list.shuffle();
-    for (var i = 0; i < 3; i++) {
+    for (var i = 0; _randomlySelectedIndexes.length < 3; i++) {
```

Comment:
too much code (and unefficient) to generate 3 random numbers.To generate a number from page N where M is the number of words per page you can use the following:

random.NextInt(M) + M × N

Reply:
[Fixed](https://github.com/breez/breezmobile/pull/180/commits/58598442b87dcde113ce27f808709656a1619364)